msm/skill_entry.py

```diff
@@ -219,15 +219,15 @@ def create_path(cls, folder, url, name=''):
 
     @staticmethod
     def extract_repo_name(url):
-        s = url.rstrip('/').split("/")[-1]
+        s = url.split("/")[-1]
```

Comment:
Isn't the rstrip a safety in case someone uses an url including the last slash, for example `https://github.com/mycroftai/mycroft-skills-manager/` ? (and the same below)

Reply:
Ah I hadn't considered that case. We should keep the rstrips in.